### PR TITLE
Add podman

### PR DESCRIPTION
**Pull request by cdpb**
Thank you for taking the time to work on a PR for Awesome-Sysadmin!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software)
- [x] Software your are submitting is not your own, unless you have a healthy ecosystem with a few contributors (which aren't your sock puppet accounts).
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.

buildah, podman, podman-compose work together under the hood but can be used independently. Because of the close relation, I submit all in one pull request. 

- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 80 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at [awesome-selfhosted](https://github.com/awesome-foss/awesome-sysadmin)
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-foss/awesome-sysadmin/issues) or [PRs](https://github.com/awesome-foss/awesome-sysadmin/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

--------------

### Why it is awesome

podman is slowly replacing docker in RHEL environments because of there daemon-less, root-less and general more secure approach - without losing compatibility to the docker eco system.

### CODE REVIEW

**Review by nodiscc — APPROVED** [review]

**Review by nodiscc — CHANGES_REQUESTED** [review]
Hi @cdbpb, to increase chances of this getting reviewed and merged, you should split the pull request (one PR for each addition).

Since https://github.com/awesome-foss/awesome-sysadmin/pull/459 we have new contribution guidelines in place, could you please take some time to answer the following questions as best you can? Thanks

- **Why is it awesome?**
- **Have you used it? For how long?**
- **Is this in a personal or professional setup?**
- **How many devices/users/services/... do you manage with it?**
- **Biggest pros/cons compared to other solutions?**

**Comment by cdpb** [comment]
- Why is it awesome?

native to use with docker cli e.g. alias docker=podman.
rootless, daemonless containers and wide support in Red Hat Products.

- Have you used it? For how long?

Yes. Since 4 years.

- Is this in a personal or professional setup?

Personal

- How many devices/users/services/... do you manage with it?

2

- Biggest pros/cons compared to other solutions?

compared to docker the biggest pro is the daemon less approach.
It does not use root for anything. It's feature rich and easy to use.
Its compatible to docker OCI

Cons:

Again, compared to docker, the daemonless approach has a con, since tools like traefik, watchtower etc are not working.

File: README.md [hunk 1]
```diff
@@ -585,13 +585,16 @@ Comparison of NoSQL servers: http://kkovacs.eu/cassandra-vs-mongodb-vs-couchdb-v
 
 *Operating system–level virtualization.*
 
+* [Buildah](https://buildah.io) - a tool that facilitates building OCI container images ([Source Code](https://github.com/containers/buildah)) `Apache-2` `Go`
```
Comment (nodiscc) [inline README.md:588]:
```suggestion
```

**Review by nodiscc — APPROVED** [review]

**Comment by cdpb** [comment]
podman-compose is just a framework on top of podman, to me they are connected.
buildah is also part of the same universe and works underneath podman. But I can submit another PR for that

**Comment by cdpb** [comment]
waiting for merge

**Pull request merged**